cogs/wordfilter/testWordFilter.py

```diff
@@ -0,0 +1,61 @@
+import re
+
+import pytest
+
+from .wordfilter import (
+    _filterWord,
+    _isAllFiltered,
+    _isOneWord,
+)
+
+
+class TestHelperFunctions:
+    @pytest.mark.parametrize(
+        ["inputPhrase", "toFilter", "result"],
+        [
+            ("I am cool", "am", "I `**` cool"),
+            (
+                "https://discord.gg/testing123",
+                r"(discord.gg\/)(?!YcW3AtX)(?!MmpqtqD)[a-zA-Z0-9]*",
+                "https://`*********************`",
+            ),
+            (
+                "https://discord.gg/YcW3AtX",
+                r"(discord.gg\/)(?!YcW3AtX)(?!MmpqtqD)[a-z,A-Z,0-9]*",
```

Comment:
Similar to [this](https://github.com/SFUAnime/Ren/pull/585#discussion_r974825559). You might just `Resolve conversation` if you want to diversify test scenarios, but even then, having 2 commas in a square-bracketed group is a nit.
```suggestion
                r"(discord.gg\/)(?!YcW3AtX)(?!MmpqtqD)[A-Za-z0-9,]*",
```

Reply:
No, I'll put this in, missed this case earlier.